vector/src/main/java/im/vector/app/features/home/room/threads/list/model/ThreadListModel.kt

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2019 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.home.room.threads.list.model
+
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import androidx.core.view.isVisible
+import com.airbnb.epoxy.EpoxyAttribute
+import com.airbnb.epoxy.EpoxyModelClass
+import im.vector.app.R
+import im.vector.app.core.epoxy.ClickListener
+import im.vector.app.core.epoxy.VectorEpoxyHolder
+import im.vector.app.core.epoxy.VectorEpoxyModel
+import im.vector.app.core.epoxy.onClick
+import im.vector.app.core.extensions.clearDrawables
+import im.vector.app.core.extensions.setLeftDrawable
+import im.vector.app.core.utils.DimensionConverter
+import im.vector.app.features.displayname.getBestName
+import im.vector.app.features.home.AvatarRenderer
+import org.matrix.android.sdk.api.session.threads.ThreadNotificationState
+import org.matrix.android.sdk.api.util.MatrixItem
+
+@EpoxyModelClass(layout = R.layout.item_thread_list)
+abstract class ThreadListModel : VectorEpoxyModel<ThreadListModel.Holder>() {
```

Comment:
We call those classes with `Item` suffix in the codebase. So maybe `ThreadItem` here.

Reply:
Ok, I thought doc suggested to end with `model` also it automatically removes the model when you building epoxy components, so `somethingModel` can be written as` something{ }`. However, yes I will change it to be aligned with the whole application naming convention